src/modules/drawing/components/DrawingExporter.vue

```diff
@@ -31,9 +31,10 @@ export default {
         },
     },
     data() {
+        const options = [new DropdownItem('KML'), new DropdownItem('GPX')]
         return {
-            exportSelection: 'KML',
-            exportOptions: [new DropdownItem('KML'), new DropdownItem('GPX')],
```

Comment:
"exportSelection" expects a String like 'KML' or 'GPX' and not a DropdownItem. I would initialize "exportSelection" with null (line 36).

Also I would not pass the "exportSelection" as "current-value" to the DropdownButton (see Line 5), as selecting a download option only triggers a download so it makes no sense for the button to be permanently highlighted after that.

Reply:
Good catch about the value type of `exportSelection`.

However I would keep it as current value, because I think the dropdown selection is IMHO the selected action that will be performed by the button if clicked. For example you only click on the dropdown arrow to see which action will be performed, but without effectively running the action.

Reply:
@jedef see last commits, I reverted this.